e2e/test/scenarios/joins/reproductions/22859-multi-nested-joins-wrong-aliasing.cy.spec.js

```diff
@@ -31,44 +31,53 @@ const questionDetails = {
 
 describe("issue 22859 - multiple levels of nesting", () => {
   beforeEach(() => {
-    cy.intercept("POST", "/api/card").as("saveQuestion");
-
     restore();
     cy.signInAsAdmin();
 
     cy.createQuestion(questionDetails, { wrapId: true, idAlias: "q1Id" });
 
     // Join Orders table with the previously saved question and save it again
-    openOrdersTable({ mode: "notebook" });
-    // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
-    cy.findByText("Join data").click();
-
-    popover().within(() => {
-      cy.findByText("Sample Database").click();
-      cy.findByText("Saved Questions").click();
-      cy.findByText(questionDetails.name).click();
+    cy.get("@q1Id").then(id => {
+      const nestedQuestionDetails = {
```

Comment:
can we move fixtures outside the test maybe?

Reply:
This is not a fixture.
But maybe I misunderstood what you wanted to say/achieve?